debian/autobake-deb.sh

```diff
@@ -77,6 +77,14 @@ disable_libfmt()
   sed '/libfmt-dev/d' -i debian/control
 }
 
+change_compat_level()
+{
+  echo "${1}" > debian/compat
+  sed -e 's/ --fail-missing//' \
+      -e 's/#override_dh_missing/override_dh_missing/' \
+      -e 's/#\tdh_missing/\tdh_missing/' -i debian/rules
```

Comment:
To make this a backwards moving to compat 11, keep the `echo` and replace with a simpler sed to remove the paragraph of `compat level 12` for when `change_compat_level 11` is specified.
```
sed -e '/compat level 12/,/^$/d' -i debian/rules
```
removes the paragraph.